http/server.ts

```diff
@@ -693,19 +618,19 @@ export async function serveTls(
  * incoming connections, and handles requests on these connections with the
  * given handler.
  *
- * If the port is omitted from the address, `:80` is used.
+ * If the port is omitted from the ListenOptions, 80 is used.
  *
- * If the host is omitted from the address, the non-routable meta-address
+ * If the host is omitted from the ListenOptions, the non-routable meta-address
```

Comment:
Consider either `Deno.ListenOptions` (aka the full type) or `config` (the name of the argument in the function signature)